wiki/Beatmap_Spotlights/pt-br.md

```diff
@@ -0,0 +1,342 @@
+---
+tags:
+  - Charts
+  - Ranking Charts
+  - Seasonal Spotlights
+---
+
+# Beatmap Spotlights
+
+![Beatmap Spotlights logo](img/logo.jpg)
+
+Os **Beatmap Spotlights** (também conhecidos como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) pela seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
```

Comment:
```suggestion
O **Beatmap Spotlights** (também conhecido como *Spotlights* ou *Ranking Charts*) é um projeto de curadoria recorrente para recomendação e destaque de [beatmaps](/wiki/Beatmaps) por seu design e gameplay excelentes e únicos. É acompanhado por uma [liga de temporada](#spotlights-seasons) que dá recompensas para todos os jogadores participantes.
```
``Beatmap spotlights`` it's just the name of one project so I think it should be singular and `pela seu design` seems strange to me, just a suggestion tho